lib/elixir/lib/integer.ex

```diff
@@ -400,17 +400,23 @@ defmodule Integer do
       iex> Integer.gcd(10, 0)
       10
 
+      iex> Integer.gcd(7, 7)
+      7
+
       iex> Integer.gcd(0, 0)
       0
 
   """
+  @spec gcd(0, 0) :: 0
+  @spec gcd(int, int) :: int when int: pos_integer
```

Comment:
There have been previous discussions about this, and long ones, So i might have gotten lost.
But does that spec say that for any pos_integer number passed as first and second argument it will return that very same pos_integer?
or it just says something different?